src/main/java/tutoraid/logic/commands/EditStudentCommand.java

```diff
@@ -30,23 +28,23 @@
 
     public static final String COMMAND_FLAG = "-s";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Edits the details of the student identified "
-            + "by the index number used in the Student Panel. "
-            + "Existing values will be overwritten by the input values.\n"
-            + "Parameters: INDEX (must be a positive integer) "
-            + PREFIX_STUDENT_NAME + "STUDENT NAME "
-            + PREFIX_STUDENT_PHONE + "STUDENT PHONE "
-            + PREFIX_PARENT_NAME + "PARENT NAME "
-            + PREFIX_PARENT_PHONE + "PARENT PHONE "
-            + "Example: " + COMMAND_WORD + " "
-            + PREFIX_STUDENT_NAME + "John Doe "
-            + PREFIX_STUDENT_PHONE + "81234567 "
-            + PREFIX_PARENT_NAME + "Mrs Doe "
-            + PREFIX_PARENT_PHONE + "91234567 ";
-
-    public static final String MESSAGE_EDIT_STUDENT_SUCCESS = "Edit successful. Displaying %s and his/her lessons.";
+    public static final String MESSAGE_USAGE = String.format("%1$s %2$s: Edits the details of the student."
+                    + "Existing values will be overwritten by the input values."
+                    + "\nParameters:"
+                    + "\nSTUDENT INDEX (must be a positive integer)"
+                    + "  %3$s[STUDENT NAME]"
```

Comment:
Not that accurate since `sn/` is part of the optional parameter
```suggestion
                    + "  [%3$sSTUDENT NAME]"
```